src/hooks/warp/api/useGetPoolsWarp.ts

```diff
@@ -0,0 +1,19 @@
+import { CyberClient } from '@cybercongress/cyber-js';
+import { QueryLiquidityPoolsResponse } from '@cybercongress/cyber-js/build/codec/tendermint/liquidity/v1beta1/query';
+import { useQuery } from '@tanstack/react-query';
+import { Option } from 'src/types';
+import { Networks } from 'src/types/networks';
+
+function useGetPoolsWarp(queryClient?: CyberClient) {
+  const { data } = useQuery({
+    queryKey: ['useFindPool', 'pools', Networks.BOSTROM],
+    queryFn: async () => {
+      return queryClient?.pools() as Option<QueryLiquidityPoolsResponse>;
```

Comment:
`queryClient!.` can be, it is ensured by `enabled` check

Reply:
<img width="684" alt="image" src="https://github.com/cybercongress/cyb-ts/assets/36644677/bfe503ee-59c4-4033-bd1e-7b977bee12f0">